ansible/README.md

```diff
@@ -149,6 +149,52 @@ You need to run `initdb` on cloudant **only once** per cloudant database to init
 **Hint:** The `initdb.yml` playbook will only initialize your database if it is not initialized already, else it will skip initialization steps.
 
 
+### Verification after Deployment
+
+After a successful deployment you can use the `wsk` CLI (located in the /bin folder of the repository) to verify that OpenWhisk is operable.
+
+1. Setup **authentication** for the CLI. This command uses the *guest* user, which is automatically added to your OpenWhisk deployment.
+
+    ```
+    cd <openwhisk_home>/bin
+    ./wsk property set --auth $(cat ../config/keys/auth.guest)
+    ```
+
+2. Setup the **API host** for the CLI.
+    * If you use **docker-machine**
+
+    ```
+    ./wsk property set --apihost $(docker-machine ip whisk)
+    ```
+    **Note**: Use your *own* docker-machine name in the command above
+
+    * If you use **vagrant**
+
+    ```
+    ./wsk property set --apihost 172.17.0.1
+    ```
+
+3. Double check the values set in the CLI.
```

Comment:
suggest you omit this - because the version and build will not match a user's output.